src/main/java/com/deque/axe/android/AxeView.java

```diff
@@ -159,6 +181,10 @@ private AxeView(
     this.text = text;
     this.viewIdResourceName = viewIdResourceName;
     this.children = children;
+    this.isWebView = isWebView;
+    this.isWebViewChild = isWebViewChild;
+
+    getDequeProps();
```

Comment:
I don't understand this line.

Reply:
This relates to your first comment. There is a list of deque props that are the calculated values. This method populates the list based on the private fields `isWebView` and `isWebViewChild`.

I agree that we should clarify that we have calculated values as dequeProps, but I prefer not prepending deque to every field in the class that fits that criteria.

Reply:
Change to initDequeProps